### ISSUE TITLE
Improve resources page

### ISSUE THREAD

**Issue by Ana06** [openSUSE/osem#1284]
As discussed in https://github.com/openSUSE/osem/pull/1261 we want to improve the resources page to make it usable.

@differentreality:

> changing the quantity of resources should be simpler/easier. If you are in a hectic registration desk and doing registrations and giving out t-shirts, you don't have time to hit edit, change the quantity in a field, hit save. So perhaps there should be a show page with a big button or something. If we want this functionality in the app, we should give some love to the UI, so that it is not just a list of things, but has a functionality that gives some added value to the organizers.

It would be nice if screenshot with ideas are provided for this issue before starting programming it, as I would say that the perfect solution is not clear and there may be some discussion about it.

**Comment by nikhilgupta1211** [comment]
@Ana06 @differentreality  as many conferences have various common resources we can give some default resources to be added like this
![image](https://cloud.githubusercontent.com/assets/14905919/23825878/4fbae798-06b8-11e7-9134-1fead27b7d0e.png)

**Comment by Ana06** [comment]
> @Ana06 @differentreality as many conferences have various common resources we can give some default resources to be added like this

The speakers are not resources, are users and we already have then in our app. And I don't think that every conference what to record the chairs, so I would leave it open, so the admins can decide what they want to track.

**Comment by differentreality** [comment]
I agree with @Ana06, we already have some stats about speakers in reports page, and I don't see why we would need to increase the number of speakers manually. While it is an interesting point, whether or not we are able to easily see how many of our speakers have already arrived (based on their registration attendance), this is not the place to do it (perhaps something to investigate and open another issue for?). 

I can see chairs being tracked for some smaller conferences/meetups but again I don't think it makes sense to have that available in advance.

Think of resources as swag (eg. tshirts, lanyards) you give to every attendee or against a donation, for which you want to know if you still have any (eg. if we offer tshirts to attendees, there is no way we can know how many size M tshirts we have left on the second day of the conference). That's a good example of what we are trying to offer the organizer in resources page. And for this case 'Tshirt M' would be 1 resource, 'Tshirt L' would be another issue, so that we can track how many we give / how many we have left every time we give one.

**Comment by nikhilgupta1211** [comment]
> @nikhilgupta1211 the speakers are not resources, are users and we already have then in our app.

@Ana06 I meant speaker as a device

**Comment by AndrewKvalheim** [comment]
Alternative proposal: #2452
